appinfo/Migrations/Version20180119080933.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace OCA\notifications\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use OCP\Migration\ISchemaMigration;
+
+class Version20180119080933 implements ISchemaMigration {
+
+	/**
+	 * @param Schema $schema
+	 * @param array $options
+	 * @throws \Doctrine\DBAL\Schema\SchemaException
+	 */
+	public function changeSchema(Schema $schema, array $options) {
+		$prefix = $options['tablePrefix'];
+		$table = $schema->getTable("${prefix}notifications");
+		$column = $table->getColumn('message');
+		$column->setNotnull(false);
+		$column = $table->getColumn('link');
```

Comment:
can we also change the reading/fetching code and make sure it consistently returns either empty string or null to avoid surprises ?

Reply:
basically addressed here - https://github.com/owncloud/notifications/pull/133/commits/2adfda98cf9eac3dcb5d662c4010125c61249171